selfdrive/car/interfaces.py

```diff
@@ -100,14 +101,14 @@ def get_pid_accel_limits(CP, current_speed, cruise_speed):
     return ACCEL_MIN, ACCEL_MAX
 
   @classmethod
-  def get_non_essential_params(cls, candidate: str):
+  def get_non_essential_params(cls, candidate: ALL_CAR):
```

Comment:
Hmm, it's technically a type annotation so it should be like `Platform`? But it's like an enum because you can do

`PLATFORM.OUTBACK`
and
`PLATFORM.COROLLA`

Reply:
I suppose both are technically correct, but functionally you will only be dealing with one `Platforms` enum at a time

Reply:
I think we should avoid using Car if we have non-car platforms. all the `CAR`s should also be renamed

Reply:
Sure, i'm saying types in python are generally not all capital to avoid confusion.